Configuration.props

```diff
@@ -37,10 +51,16 @@
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
   </PropertyGroup>
   <PropertyGroup>
+    <_MingwPrefixTail Condition=" '$(HostOS)' == 'Darwin' ">.static</_MingwPrefixTail>
+    <_MingwCommandPrefix32>i686-w64-mingw32$(_MingwPrefixTail)</_MingwCommandPrefix32>
```

Comment:
Right, but where should this be put? It is used by more than one _target and is not really dependent on the OS, so it doesn't belong in `Configuration.OperatingSystem.props` either. It should also not be customized in the overrides file (nothing prevents it, of course) and the underscore marks it as private by convention